bin/propolis-standalone/Cargo.toml

```diff
@@ -17,14 +17,17 @@ libc.workspace = true
 toml.workspace = true
 tokio = { workspace = true, features = ["io-util", "rt-multi-thread"] }
 serde = { workspace = true, features = ["derive"] }
-propolis.workspace = true
+propolis = { workspace = true, features = ["crucible-full"], default-features = false }
```

Comment:
I'd like to keep crucible as an optional dependency for those not using it in their standalone work.

Reply:
ah, my brain missed the feature flags at the bottom of the toml. can do